packages/ag-charts-types/src/chart/labelOptions.ts

```diff
@@ -15,7 +15,7 @@ export interface AgChartLabelOptions<TDatum, TParams> extends Toggleable, FontOp
     formatter?: Formatter<AgChartLabelFormatterParams<TDatum> & TParams>;
 }
 
-export interface AgChartLabelFormatterParams<TDatum> extends AgChartCallbackParams<TDatum> {
+export interface AgChartLabelFormatterParams<TDatum = any> extends AgChartCallbackParams<TDatum> {
```

Comment:
Preferably we don't do this so we remember to type them. Why was this needed?

Reply:
The grid callbacks don't force users to type them, this is to align with the current convention

Reply:
Reverted